main.cpp

```diff
@@ -294,6 +297,18 @@ void Computer::changeNumOperatorTypes(int newNumOperatorTypes)
     std::cout << "New number of operator types: " << language1.numOperatorTypes << std::endl;
 }
 
+void Computer::changeNumDataTypesThis(int numDataTypes)
+{
+    this -> language1.numDataTypes = numDataTypes;
+    std::cout << "New number of data types: " << language1.numDataTypes << std::endl;
+}
+
+void Computer::changeNumOperatorTypesThis(int numOperatorTypes)
+{
+    this -> language1.numOperatorTypes = numOperatorTypes;
+    std::cout << "New number of operator types: " << language1.numOperatorTypes << std::endl;
+}
```

Comment:
don't put spaces around the `->` operator.
your std::cout statements need to show usage of `this->`.
- [ ] revise